charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -110,6 +111,23 @@ spec:
                         rootDirectory:
                           type: string
                           minLength: 1
+                    gcs:
```

Comment:
If you want to enforce what kind of configurations are valid for `gcs`, you can do something like this:
```
gcs:
  type: object
  properties:
  ...
  oneOf:
  - required:
    - createSecret
    - serviceAccountJSON
    properties:
      createSecret:
        enum:
        - false
    allOf:
    - not:
        required:
        - secretName
  - required:
    - secretName
    allOf:
    # whatever fields you want to restrict
    ...
```
This would allow two valid sub-schemas,
```
type: gcs
gcs:
  createSecret: false
  bucket: ...
  serviceAccountKeyJSON: ...
```
and:
```
type: gcs
gcs:
  bucket: ...
  secretName: ...
```